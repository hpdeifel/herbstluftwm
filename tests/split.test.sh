#!/bin/bash

${HERBSTCLIENT} split left
${HERBSTCLIENT} dump
${HERBSTCLIENT} quit
