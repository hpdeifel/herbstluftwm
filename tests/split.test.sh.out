(split horizontal:0.5:1 (clients vertical:0) (clients vertical:0))
